README.md

```diff
@@ -13,6 +13,28 @@ IRC channel: https://freenode.net #googlebenchmark
 
 [Additional Tooling Documentation](docs/tools.md)
 
+## Building
+
+The basic steps for configuring and building the library look like this:
+
+```bash
+$ git clone https://github.com/google/benchmark.git
+# Benchmark requires GTest as a dependency. Add the source tree as a subdirectory.
+$ git clone https://github.com/google/googletest.git benchmark/googletest
```

Comment:
I'd **highly** recommend
1. providing an option specifying the path to the gtest sources
2. defaulting to `/usr/src/googletest/`, since that is the natural location, and is where debian puts it.

Reply:
First, I don't really care to allow users to specify random/arbirtrary out-of-tree GTest source directories. We already have a mechanism to allow users to specify arbitrary GTest installation using `CMAKE_MODULE_PATH` to influence `find_package(GTest)`.

I've changed the build, as you've suggested, to only download dependencies if `-DBENCHMARK_DOWNLOAD_DEPENDENCIES=ON` is specified. So GTest installations among `/usr/` and `/usr/local/` should work by default.

Reply:
why not git submodules?